src/components/ComponentList.js

```diff
@@ -164,13 +166,24 @@ export default class ComponentList extends React.Component {
   }
 
   render() {
-    const { loadMoreRows, listHeight, noRowsRenderer, list, fetchingRenderer, renderFilterBar } = this.props
+    const {
+      loadMoreRows,
+      listHeight,
+      noRowsRenderer,
+      list,
+      isFetching,
+      headers,
```

Comment:
should raise the level of abstraction here. Rather than passing in `headers`, pass in the one value we need from the headers, `hasMore`.